armi/reactor/components/complexShapes.py

```diff
@@ -230,13 +237,18 @@ def __init__(
         )
 
     def getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
-        return self.getDimension("od", Tc, cold) + self.getDimension(
-            "helixDiameter", Tc, cold=cold
+        """the diameter of a circle which is encompassed by the exterior of the wire-wrap"""
+        return self.getDimension("helixDiameter", Tc, cold=cold) + self.getDimension(
+            "od", Tc, cold
         )
 
     def getCircleInnerDiameter(self, Tc=None, cold=False):
-        return self.getDimension("id", Tc, cold) + self.getDimension(
-            "helixDiameter", Tc, cold=cold
+        """the diameter of a circle which is encompassed by the interior of the wire-wrap
+
+        - should be equal to the outer diameter of the pin in which the wire is wrapped around
+        """
+        return self.getDimension("helixDiameter", Tc, cold=cold) - self.getDimension(
+            "od", Tc, cold
```

Comment:
Why would we be using the outer diameter here, if the method is called "getCircleInnerDiameter"?

This is not clear to me.

Reply:
its because a helix is weird. getCircleInnerDiameter is supposed to be the circle the component would enclose. For a helix, to enclose means to wrap around rather than looking at the anulus of the helix wire.

Reply:
In the context of a Helix, the outer diameter (`od`) is the diameter of the helical wire wrap. In order to get the diameter of the inner circle of the helix, which is bounded by the interior of the wire wrap, we start from the `helixDiameter` and increment inwards. The opposite is the case for `getBoundingCircleOuterDiameter`. 

For Helixes (Helixi?) the inner diameter, `id`, is the inner diameter of the wire wrap. So before we were returning the `getCircleInnerDiameter` incorrectly - for 99% of the time when the `id = 0.0`, the `getCircleInnerDiameter` was returning the `helixDiameter` which isn't quite right. 

The bullet points added to the Notes section of the class docstring try to address this. 

@onufer

Reply:
okay, then